pkg/schedule/operator/operator_controller.go

```diff
@@ -602,7 +602,7 @@ func (oc *Controller) removeOperatorLocked(op *Operator) bool {
 	regionID := op.RegionID()
 	if cur := oc.operators[regionID]; cur == op {
 		delete(oc.operators, regionID)
-		oc.updateCounts(oc.operators)
+		oc.counts[op.SchedulerKind()]--
```

Comment:
Why not removed `updateCounts` from `removeOperatorsLocked` and remove function `updateCounts` directly?

Reply:
done